helm-charts/FATE/templates/core/python-spark.yaml

```diff
@@ -129,11 +129,6 @@ spec:
                 cp /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults-template.conf /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults.conf
                 sed -i "s/fateflow/${POD_IP}/g" /data/projects/spark-2.4.1-bin-hadoop2.7/conf/spark-defaults.conf
                 
-                # logs
-                mkdir -p /data/projects/fate/fateflow/logs/fate_flow/
-                touch /data/projects/fate/fateflow/logs/fate_flow/INFO.log
-                ln -sf /dev/stdout /data/projects/fate/fateflow/logs/fate_flow/INFO.log
-                
```

Comment:
https://github.com/FederatedAI/KubeFATE/commit/772e174dba04b161c493e8b8a844bdd9b371cb1c

It's you who removed these stuffs on May 10 and now you add them back, what is the story behind this?

Reply:
it means it is removed. i do not add them back.

Reply:
Oops, my fault